proto/one/common/core/content.proto

```diff
@@ -118,6 +119,9 @@ message Content {
         UnitAgencyCodeTypes unitAgencyCodeTypes = 77;
         QuantityTypes quantityTypes = 81;
 
+        // model
+        ModelInputs modelInputs = 104;
```

Comment:
Since you changed the name this is not going to compile, will need to change it here too